src/app/medInria/medLogger.cpp

```diff
@@ -88,10 +88,6 @@ medLogger::medLogger() : d(new medLoggerPrivate)
     dtkLogger::instance().attachFile(dtkLogPath(qApp));
     dtkLogger::instance().attachConsole();
 
-    // Redirect cerr and cout messages
-    dtkLogger::instance().redirectCerr();
```

Comment:
not needed anymore?

Reply:
Questionable, as said in the PR summary.
To me, not needed in any code in the universe

Reply:
not needed anymore thanks to https://github.com/medInria/medInria-public/pull/703